app/javascript/packs/style.jsx

```diff
@@ -40,7 +40,6 @@ export const Dialogue = styled.div`
     props.save &&
     `
     width: 550px;
-    height: 350px;
```

Comment:
is this intentional? seams unrelated

Reply:
Has to do with the height of the panel when removing the warning about 0.58
https://github.com/brave-intl/publishers/issues/2113